ts/app.ts

```diff
@@ -1,9 +1,5 @@
-import {
-  ComponentAnnotation as Component,
-  ViewAnnotation as View,
-  bootstrap,
-  For
-} from 'angular2/angular2';
+/// <reference path="../typings/angular2/angular2.d.ts" />
+import {Component, View, bootstrap, For} from 'angular2/angular2';
```

Comment:
Don't know how to squash into a single commit. You can still clone my fork if you can't wait @mgechev to merge